pulsar-broker/src/test/java/org/apache/pulsar/broker/service/OneWayReplicatorTest.java

```diff
@@ -698,10 +708,19 @@ public void testDeleteNonPartitionedTopic() throws Exception {
         // Verify replicator works.
         verifyReplicationWorks(topicName);
 
+        admin1.topicPolicies().setCompactionThreshold(topicName, 1000_000);
+        Awaitility.await().atMost(Duration.ofSeconds(3600)).untilAsserted(() -> {
+            assertEquals(admin2.topicPolicies().getCompactionThreshold(topicName), 1000_000);
+        });
+
         // Disable replication.
         setTopicLevelClusters(topicName, Arrays.asList(cluster1), admin1, pulsar1);
         setTopicLevelClusters(topicName, Arrays.asList(cluster2), admin2, pulsar2);
 
+        Thread.sleep(3 * 1000);
```

Comment:
Why do we sleep 3s here?

Reply:
Because the operation that calls the expanding API will be executed aync

Reply:
Should we use `Awaitility` to verify the condition?